test/lib/import/brca/providers/birmingham/birmingham_handler_newformat_test.rb

```diff
@@ -58,6 +58,30 @@ def setup
     assert_equal 1, genotypes[1].attribute_map['teststatus']
   end
 
+  test 'process_mutation_from_fullscreen_newformat' do
+    @genotype.attribute_map['genetictestscope'] = 'Full screen BRCA1 and BRCA2'
+    @record.raw_fields['overall2'] = 'UV'
+    processor = variant_processor_for(@record)
+    genotypes = processor.process_variants_from_report
+    assert_equal 8, genotypes[0].attribute_map['gene']
+    assert_equal 7, genotypes[1].attribute_map['gene']
+    assert_equal 2, genotypes[1].attribute_map['teststatus']
+    assert_equal 3, genotypes[1].attribute_map['variantpathclass']
+    assert_equal 'p.Thr1256Argfsx', genotypes[1].attribute_map['proteinimpact']
+  end
+
+  test 'negative_tests_from_fullscreen_newformat' do
+    negative_record_newformat = build_raw_record('pseudo_id1' => 'bob')
+    negative_record_newformat.raw_fields['overall2'] = 'Normal'
```

Comment:
Could this be including in the `build_raw_record` call above? There are a few other examples.

Reply:
`build_raw_record` creates a skeleton structure , for which we change the attributes as per the test case requirements and then do the assertions. Same is being used above too to build a negative record of new format.